### DISCUSSION TITLE
Apple PAGES crashes when trying to insert an image

### DISCUSSION THREAD

**Discussion by Roger-76** [community/community#55843] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

Hi! all Github's developpers,
I'm very new user and don't know how to post questions / problems.
I'm using a mid-2012 13" MacBook Pro (model A1278), using Open Legacy Patcher 0.65 and Ventura 13.2 without any problem.
But when installing Ventura 13.3 or 13.4, Apple Pages ( v12.2 and v13.x) crashes when trying to insert jpg or other format image.
Can somebody help ?

Many thanks.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @Roger-76 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this Apple and their programs. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
